### ISSUE TITLE
Support for "pinch to zoom" on the mouse wheel

### ISSUE THREAD

**Issue by Elger9** [waydroid/waydroid#1087]
### Is your feature request related to a problem? Please describe.

Without 'pinch to zoom' it is difficult or even impossible to use many applications.

### Describe the solution you'd like.

A 'pinch to zoom' simulation when turning the mouse wheel, although imperfect, would be sufficient in many cases.

### Describe alternatives you've considered.

The feature can be done in a number of other ways, e.g. as in Bluestacks:
- Hold the Ctrl key while pressing the – key to zoom out and the + key to zoom in
- Hold the Ctrl key while scrolling up or down using the mouse wheel

**Comment by FordCorn** [comment]
+1

**Comment by kaarelen** [comment]
+1

**Comment by wdebie** [comment]
+1

**Comment by m0nclous** [comment]
+1

**Comment by thomasfinstad** [comment]
I believe a modifier + scroll function would be useful, that way we can choose to keep scroll function and use  ctrl+scroll or meta+scroll etc for zoom. The app I installed waydroid for is very hard / impossible to use without the ability to zoom in and out.

**Comment by DumbledorBoB** [comment]
On a game called Townsmen using Bluestacks/Windows, it uses the keyboard's Up and Down arrow keys to zoom in and out. Might that be a possibility?
(e.g. Desktop users usually do not have a trackpad but all keyboards have the arrow keys)

**Comment by NanamiTV** [comment]
i agree with this, im trying to play the simpsons tapped out, but since the game loads in fully zoomed-in theres no way for me to actually get a good view of whats going on. tried everything i found online but nothing works.
